lib/puppet/provider/jenkins_credentials/cli.rb

```diff
@@ -108,7 +108,7 @@ def self.credentials_list_json(catalog = nil)
     begin
       JSON.parse(raw)
     rescue JSON::ParserError
-      raise Puppet::Error, "unable to parse as JSON: #{raw}"
+      raise Puppet::Error, "unable to parse as JSON: {REDACTED}"
```

Comment:
I wonder if we should instead show the error. Capture the exception message and show that. Otherwise rephrase it as:
```suggestion
      raise Puppet::Error, 'Unable to parse Jenkins credentials list as JSON'
```